include/index.h

```diff
@@ -369,18 +372,14 @@ template <typename T, typename TagT = uint32_t, typename LabelT = uint32_t> clas
     // Filter Support
 
     bool _filtered_index = false;
-    // Location to label is only updated during insert_point(), all other reads are protected by
```

Comment:
insert_point should take string formatted labels (since exposed to user). conversion to label type should happen internally